docs/src/installation.md

```diff
@@ -1,316 +0,0 @@
----
-id: installation
-title: "Installation"
----
-
-<!-- TOC -->
-
-### Prerequisites for .NET
```

Comment:
If I see this correctly, this part was fully nuked out of the docs. So there is no mentioning anywhere anymore about the Playwright .NET CLI tool thingy.

Reply:
Yeah, let me restore it...